src/main/java/exercise9th/mybatisdemo/NameController.java

```diff
@@ -0,0 +1,25 @@
+package exercise9th.mybatisdemo;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class NameController {
+
+    private NameMapper nameMapper;
+
+
+    public NameController(NameMapper nameMapper) {
+        this.nameMapper = nameMapper;
+    }
+
+    @GetMapping("/names")
+    public List<Name> getNames(){
+
+        List<Name> names = nameMapper.findAll();
+
+        return names;
```

Comment:
授業動画に合わせてこのようにしてくれているとは思いますが、下記のように書いてOKです。
```suggestion
        return nameMapper.findAll();
```

Reply:
ご助言のとおり更新しました。
![image](https://user-images.githubusercontent.com/113277395/201450220-04884e8b-267c-41e5-be56-fa499793e602.png)

Reply:
ありがとうございます！問題ないです！